packages/screens/Wallet/Screens/NativeWallet.tsx

```diff
@@ -43,33 +45,47 @@ const NativeWallet: ScreenFC<"NativeWallet"> = () => {
             alignItems: "center",
           }}
         >
-          <CustomButton
-            title="Create Wallet"
-            onPress={(_, navigation) => {
-              navigation.navigate("CreateWallet");
-            }}
-          />
+          {
+            // disable multiple wallet creation, remove to re enable multiple
+            !nativeWallet && (
+              <CustomButton
+                title="Create Wallet"
+                onPress={(_, navigation) => {
+                  navigation.navigate("CreateWallet");
+                }}
+              />
+            )
+          }
           {nativeWallet && (
             <CustomButton
               title="Continue"
               style={{
                 marginVertical: layout.spacing_x2,
               }}
               onPress={(_, navigation) => {
-                navigation.navigate("MiniTabs", { screen: "MiniChats" });
+                if (appMode === "mini") {
```

Comment:
should we create enum AppMode instead of string ?

Reply:
an Enum doesn't make sense. The comparison is always against === "mini".
What I would make sense is to refactor this code everywhere and replace it with something like:
`useIsMiniMode()`